doc/how_to_guides/pick_ik/pick_ik_tutorial.rst

```diff
@@ -0,0 +1,130 @@
+pick_ik Kinematics Solver
+=========================
+
+``pick_ik`` is an inverse kinematics (IK) solver compatible with MoveIt 2,
+developed by PickNik Robotics. It is designed to provide robust and customizable IK solutions,
+offering a wide range of features.
+
+The solver in ``pick_ik`` is a reimplementation of ``bio_ik``, integrating a global optimizer and a local optimizer.
+The global optimizer utilizes evolutionary algorithms to explore alternative solutions within the solution space
+and identify global optima. Building upon the results obtained by the global optimizer, the local optimizer applies
+gradient descent for iterative refinement of the solution. It takes a potential optimum solution provided by the global
+optimizer as input and aims to improve its accuracy and ultimately convergence to the most optimum solution.
+
+Getting Started
+---------------
+Before proceeding, please ensure that you have completed the steps outlined in the :doc:`Getting Started Guide </doc/tutorials/getting_started/getting_started>`.
+
+Additionally, it is required to have a MoveIt configuration package specifically tailored to your robot.
+This package can be created using the :doc:`MoveIt Setup Assistant </doc/examples/setup_assistant/setup_assistant_tutorial>`.
+
+Installation
+------------
+
+From binaries
+^^^^^^^^^^^^^
+Make sure your ROS2 installation is sourced and run the following command ::
+
+    sudo apt install ros-$ROS_DISTRO-pick-ik
+
+From source
+^^^^^^^^^^^
+
+Create a colcon workspace. ::
+
+    export COLCON_WS=~/ws_moveit2/
+    mkdir -p $COLCON_WS/src
+
+Clone this repository in the src directory of your workspace. ::
+
+    cd $COLCON_WS/src
+    git clone -b main https://github.com/PickNikRobotics/pick_ik.git
+
+Set up colcon mixins. ::
+
+    sudo apt install python3-colcon-common-extensions
+    sudo apt install python3-colcon-mixin
+    colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+    colcon mixin update default
+
+Build the workspace. ::
+
+    cd /path/to/your/workspace
+    colcon build --mixin release
+
+Usage
+-----
+
+Using pick_ik as a Kinematics Plugin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can use :doc:`MoveIt Setup Assistant </doc/examples/setup_assistant/setup_assistant_tutorial>` to create
+the configuration files for your robot to use it with MoveIt, or edit the ``kinematics.yaml`` file in your
+robot's config directory to use pick_ik as the IK solver. ::
```

Comment:
I would say you can either edit the `kinematics.yaml` file, or you can create a new one and make sure it's referenced in the launch config. The `MoveItConfigsBuilder` allows you to specify files with other names in case you want to have multiple configs around.

Reply:
I'd even suggest going as far as creating your own launch file similar to to the Quickstart tutorial, except using Pick IK with your own `kinematics.yaml` file such that readers can quickly start up an example with Pick IK.

Reply:
Thanks for your comment! I submitted a [PR](https://github.com/ros-planning/moveit_resources/pull/180) with two newly added files `kinematics_pick_ik.yaml` and `demo_pick_ik.launh.py` files under the [moveit_resources/panda_moveit_config](https://github.com/ros-planning/moveit_resources/tree/ros2/panda_moveit_config) pkg. The aim is to make it easier for users to launch a tutorial with `pick_ik` configured as the solver.

Reply:
After getting feedback on the PR, I will update the `pick_ik` tutorial. Thanks again!